shed/schema.go

```diff
@@ -114,6 +114,7 @@ func (db *DB) schemaIndexPrefix(name string) (id byte, err error) {
 
 // getSchema retrieves the complete schema from
 // the database.
+// This is exported for displaying the schema info outside
```

Comment:
remove the comment - the function is not exported

Reply:
Done